packages/fields/src/types/OEmbed/OEmbedBlock.js

```diff
@@ -51,7 +51,16 @@ export class OEmbedBlock extends Block {
               'A reference back to the Slate.js Serialised Document this embed is contained within',
           },
         },
-        access: listConfig.listAccess,
+        access: Object.entries(listConfig.listAccess).reduce(
```

Comment:
Util function?

Reply:
It's now used 3 times?

Reply:
4 times!

Reply:
Yeah, maybe, but also I don't have time right now :-)

Reply:
The real refactor should come one level higher with the way we create these Aux lists for blocks, but that's future Tim's problem.